region/client.go

```diff
@@ -464,6 +465,20 @@ func (c *client) receiveRPCs() {
 	}
 }
 
+func (c *client) trackRPCResult(rpc hrpc.Call, status string) {
+	if m, ok := rpc.(*multi); ok {
+		for _, call := range m.calls {
+			if call != nil {
+				rpcResultCount.WithLabelValues(
+					c.Addr(), call.Name(), status, "batch").Inc()
```

Comment:
For each call these labels are going to be the same all the time, call.Name() is going to be "Mutate" though technically could be a "Get".

I'd rather we do a single:
`rpcResultCount.WithLabelValues(c.Addr(), "Mutate", status, "batch").Add(len(m.calls))`

Because the WithLabelValues calls can be expensive.

To be more correct you could add up all the Mutate and the Get calls in a loop and then do the `Add` for each of them.